packages/e2e-tests/puppeteer/__tests__/web3torrent.test.ts

```diff
@@ -55,6 +67,18 @@ describe('Supports torrenting among peers with channels', () => {
     }
   });
 
+  if (USES_VIRTUAL_FUNDING) {
```

Comment:
Does anyone have an opinion on how we organize these tests?

Reply:
Given that the tests take some time to run, it might be worth running tests in multiple files, in parallel. And bump the machine's resources to large